editor/editor_node.cpp

```diff
@@ -4144,6 +4144,9 @@ Ref<ImageTexture> EditorNode::_load_custom_class_icon(const String &p_path) cons
 		Ref<Image> img = memnew(Image);
 		Error err = ImageLoader::load_image(p_path, img);
 		if (err == OK) {
+			if (img->get_size() > Size2(32, 32)) { // If possible, lets turn this into a non-magic number.
```

Comment:
I don't think using a constant is necessary here. We don't have a strict policy against using magic numbers, especially if they are only used once.

Reply:
In that case, I'll remove the comment.